server/server.go

```diff
@@ -39,8 +39,11 @@ import (
 	"github.com/temporalio/web-go/server/server_options"
 )
 
-// // go:embed generated/ui
-// var uiAssets embed.FS
+// TODO resolving one of these two issues may allow to clean the embed string
```

Comment:
until the TODO issues resolved, used this script to generate the embed string

used this to generate the paths
``` shellscript
UI_OUT=./server/generated/ui
UI_HIDDEN_FILES=$(find $UI_OUT -name "_*")
UI_HIDDEN_FILE_DIRS=$(dirname $UI_HIDDEN_FILES)
echo `echo "${UI_HIDDEN_FILE_DIRS[@]}" | sort | uniq `
```